rules-generated/quarkus/hibernate/hibernate-to-quarkus.windup.xml

```diff
@@ -0,0 +1,148 @@
+                    <link title="Quarkus - Guide" href="https://quarkus.io/guides/hibernate-orm#envers" />
+                </hint>
+            </perform>
+        </rule>
+        <rule id="quarkus-hibernate-envers-00010">
+            <when>
+                <dependency groupId="org.hibernate" artifactId="hibernate-envers"/>
+            </when>
+            <perform>
+                <hint title="Replace the 'hibernate-envers' dependency with Quarkus 'quarkus-hibernate-envers' extension" effort="1" category-id="mandatory">
+                    <message>
+                        Replace the `org.hibernate:hibernate-envers` dependency with the Quarkus dependency `io.quarkus:quarkus-hibernate-envers`  
+                        Further information in the link below.
+                    </message>
+                    <link title="Quarkus - Guide" href="https://quarkus.io/guides/hibernate-orm#envers" />
+                </hint>
+            </perform>
+        </rule>
+        <rule id="quarkus-hibernate-envers-00020">
+            <when>
+                <javaclass references="org.hibernate.envers.{fileName}" >
+                    <location>IMPORT</location>
+                </javaclass>
+            </when>
+            <perform>
+                <hint title="Replace the 'hibernate-envers' dependency with Quarkus 'quarkus-hibernate-envers' extension" effort="1" category-id="mandatory">
+                    <message>
+                        Replace the `org.hibernate:hibernate-envers` dependency with the Quarkus dependency `io.quarkus:quarkus-hibernate-envers`  
+                        Further information in the link below.
+                    </message>
+                    <link title="Quarkus - Guide" href="https://quarkus.io/guides/hibernate-orm#envers" />
+                </hint>
+            </perform>
+            <where param="fileName">
+                <!-- from https://docs.oracle.com/javase/specs/jls/se11/html/jls-3.html#jls-3.8 -->
+                <matches pattern="[a-zA-Z_$*][a-zA-Z0-9_$]*" />
+            </where>
+        </rule>
+        <rule id="quarkus-hibernate-reactive-00000">
+            <when>
+                <project>
+                    <artifact groupId="org.hibernate.reactive" artifactId="hibernate-reactive-core"/>
+                </project>
+            </when>
+            <perform>
+                <hint title="Replace the 'hibernate-reactive-core' dependency with Quarkus 'quarkus-hibernate-reactive' extension" effort="1" category-id="mandatory">
+                    <message>
+                        Replace the `org.hibernate.reactive:hibernate-reactive-core` dependency with the Quarkus dependency `io.quarkus:quarkus-hibernate-reactive`  
+                        Further information in the link below.
+                    </message>
+                    <link title="Quarkus - Guide" href="https://github.com/quarkusio/quarkus-quickstarts/tree/master/hibernate-reactive-quickstart" />
```

Comment:
Good catch: let me put in place something to have `Quarkus - Guide` as default when using a guide and `Quarkus - Quickstart` otherwise. I know it's not a "fully and always" working approach but right now we just have links for guide and quickstarts for enough for current approach.

Reply:
Fixed in commit  a44561d